tox.ini

```diff
@@ -10,13 +10,15 @@ python =
 envlist =
     flake8,
     py{37,38,39,310}-drf3-django{22,32}
-    py{38,39,310,311}-drf3-django{40}
+    py{38,39,310,311}-drf3-django{40,41,42}
```

Comment:
Apparently, according to [this](https://docs.djangoproject.com/en/4.2/faq/install/), `django 4.0` doesn't support `python 3.11`. I think we should separate it like this:
```
py{38,39,310}-drf3-django40
py{38,39,310,311}-drf3-django{41,42}
```
Do you agree?

Reply:
@KaratasFurkan I agree. I pushed to my branch (and then even force pushed) but I don't see the UI updating for this MR? 🤔 

You can see the commit for the branch here: https://github.com/Hipo/drf-extra-fields/compare/master...johnthagen:drf-extra-fields:django-4.2

Reply:
Worked! :tada: